tests/testplotinstantiation.py

```diff
@@ -23,6 +24,11 @@ class TestPlotInstantiation(ComparisonTestCase):
     def setUp(self):
         if pyplot is None:
             raise SkipTest("Matplotlib required to test plot instantiation")
+        self.default_comm = renderer.comms['default']
+        renderer.comms['default'] = JupyterPushComms
+
+    def teardown(self):
+        renderer.comms['default'] = self.default_comm
 
```

Comment:
My last comment is that it might be nice to have as many unit tests as possible. I realize it is tricky without the notebook and Javascript but maybe the decode method can be tested (if made into a classmethod). Hopefully, ``__init__`` can also be tested in some cases...